site/bn.njk

```diff
@@ -0,0 +1,118 @@
+                <li class="pb-2">ক্লাবের সভা, ধর্মীয় পরিষেবা এবং ব্যক্তিগত দলগুলির মতো সামাজিক সমাবেশ এবং ইভেন্টগুলি থেকে দূরে থাকুন।</li>
+                <li class="pb-2">আপনার ভ্রমণের পরিমাণ সর্বনিম্ন হ্রাস করুন। একেবারে প্রয়োজনীয় না হলে দীর্ঘ দূরত্বে ভ্রমণ করবেন না।</li>
+                <li class="pb-2">একেবারে প্রয়োজনীয় না হলে পাবলিক ট্রান্সপোর্ট ব্যবহার করবেন না।</li>
+                <li class="pb-2">আপনি যদি বাড়ি থেকে কাজ করতে পারেন তবে বাড়ি থেকে কাজ করুন। প্রয়োজনে আপনার নিয়োগকর্তাকে দূরবর্তী কাজের অনুমতি দেওয়ার জন্য অনুরোধ করুন।</li>
+                <li class="pb-2">ফোন কল বা ভিডিও চ্যাটের মতো দূরবর্তী বিকল্পের সাথে যতটা সম্ভব সামাজিক যোগাযোগের স্থান পরিবর্তন করুন।</li>
+                <li class="pb-2">একেবারে প্রয়োজনীয় না হলে আপনার বাড়ি ছেড়ে যাবেন না।</li>
+            </ol>
+
+            <p class="text-xl">
+                দয়া করে মনে রাখবেন কোনও সঠিক বা ভুল পরিমাণ পদক্ষেপ নিতে হবে না। আপনি যে পদক্ষেপে স্বাচ্ছন্দ্য বোধ করেন কেবল সেগুলি করুন এবং জীবনযাত্রা কে হুমকির মুখে ফেলবেন না. <strong>চাকরী ছেড়ে দেওয়ারও দরকার নেই!</strong>
+                যাহোক, মনে রাখবেন যে প্রত্যেকটা পদক্ষেপই কার্যকর।
+            </p>
+
+            <h2 class="text-3xl">কেন এটি এত গুরুত্বপূর্ণ</h2>
+
+            <p>
+                SARS-CoV-2 একটি অত্যন্ত সংক্রামক এবং মারাত্মক ভাইরাস যা COVID-19 নামে একটি শ্বাসযন্ত্রের রোগের কারণ।
+                আপনি এটি 2019-nCoV, নভেল করোনাভাইরাস, উহান করোনাভাইরাস, চায়না বা উহান ফ্লু বা কেবলমাত্র করোনাভাইরাস সহ এর অন্যান্য অনেক নামের একটির অধীনে জানেন। এই সমস্তগুলি একই ভাইরাসকে বোঝায় যা আমাদের এই আন্দোলনটি থামানোর চেষ্টা করছে।
+            </p>
+
+            <p>
+               গত মাস এবং সপ্তাহগুলিতে, এই ভাইরাস এবং তার সাথে সম্পর্কিত রোগটি ইনফ্লুয়েঞ্জা ভাইরাস এবং মৌসুমী ফ্লু এর সাথে বারবার তুলনা করা হচ্ছে। লক্ষণগুলিতে মিল থাকার কারণে এবং আপাতদৃষ্টিতে একইরকম তীব্রতার কারণে এই তুলনাটি অনেক লোকের কাছে স্বাভাবিকভাবেই আসে তবে যাইহোক, আমরা এই নতুন ভাইরাস সম্পর্কে এখন অবধি যা জানি, তা সমাজকে যে হুমকির মুখোমুখি করে তা সহজেই সরানো যায় না।
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12 font-bold">
+                <li class="pb-2">
+                    <h3>ফ্লুর চেয়ে আরও ছোঁয়াচে/h3>
+                    <p class="font-normal">
+                        1.4 - 6.49 এর মধ্যে একটি গড়  R0 অনুমান 3.28 <a class="source-link" href="https://academic.oup.com/jtm/advance-article/doi/10.1093/jtm/taaa021/5735319" target="_blank">[1]</a></sup> এর গড় অনুমানের সাথে, SARS-CoV-2 অনেক বেশি সংক্রামক এবং মৌসুমী ফ্লুর তুলনায় অনেক দ্রুত ছড়িয়ে পড়ে, যার একটি মধ্যম R0 রয়েছে 1.28<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/25186370" target="_blank">[2]</a></sup>।
+
+                        With an estimated R0 (the expected number of cases directly generated by one case in a population where all individuals are susceptible to infection) between 1.4 &ndash; 6.49 and a mean estimate of 3.28<sup>,
```

Comment:
Hi @shovoncse, the translation looks good so far, but seems to be incomplete at this point?

Reply:
Hello @phelmig , Thank you for the suggestion. I didn't notice that point. Please check the current status. Thanks